app/components/RandomQuote/RandomQuote.css

```diff
@@ -20,3 +20,8 @@
 .fetchNew {
   margin-bottom: 5px;
 }
+
+.rotateIcon {
+  transform: rotate(360deg);
+  transition: 1s;
```

Comment:
I feel like a `cubic-bezier` would be very fitting, in which it speeds up at the middle and slows down at the end. 

On that note, we should standardize a `cubic-bezier` for the webapp as a variable. I feel like animations across the webapp is very "random", with different animations, speeds and lengths. I'll write it up as a todo, so you can keep it like this if you want.

Reply:
I thought so too, but the animation was just sooo satisfying by default so I saw no reason to create more code for little to no benefit.

Reply:
Yea, I'll agree that it looks smooth